bundles/org.openhab.binding.visualcrossing/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -1969,6 +1989,10 @@
 			potential energy (CAPE) and convective inhibition (CIN), predicted rain and wind. Typically, a severe risk value less
 			than 30 indicates a low risk, between 30 and 70 a moderate risk and above 70 a high risk.
 		</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>Info</tag>
```

Comment:
What is `status` and `info` tags?

Reply:
https://next.openhab.org/docs/developer/bindings/thing-xml.html#general-information-about-tags
And more specific:
https://next.openhab.org/docs/developer/bindings/thing-xml.html#tagging-conventions-for-commonly-confused-use-cases => point 8

Reply:
Umm. I understand where you are coming from. However I have several pedantic issues with your proposal..

Strictly Measurement is a real time measure of an actual physical phenomenon. e.g. Measurement+Temperature). By contrast your channel is more a future warning. Also "risk" is not really a physical measurement but probably rather some kind of algorithmic weighted risk assessment. And finally storms are not so much characterised by precipitation as by wind. So my counter suggestion would be Forecast+Wind. => Please let me know what you think?

Reply:
Good for me